appintro/build.gradle.kts

```diff
@@ -82,35 +83,54 @@ publishing {
         register<MavenPublication>("release") {
             afterEvaluate {
                 from(components["release"])
-            }
-            pom {
-                name.set(POM_NAME)
-                description.set(POM_DESCRIPTION)
-                url.set(POM_URL)
-                licenses {
-                    license {
-                        name.set(POM_LICENSE_NAME)
-                        url.set(POM_LICENSE_URL)
-                    }
-                }
-                scm {
-                    connection.set(POM_SCM_CONNECTION)
-                    developerConnection.set(POM_SCM_CONNECTION)
+
+                pom {
+                    name.set(POM_NAME)
+                    description.set(POM_DESCRIPTION)
                     url.set(POM_URL)
-                }
-                developers {
-                    developer {
-                        id.set("paolorotolo")
-                        name.set("Paolo Rotolo")
-                        email.set("paolo@rotolo.dev")
+                    licenses {
+                        license {
+                            name.set(POM_LICENSE_NAME)
+                            url.set(POM_LICENSE_URL)
+                        }
+                    }
+                    scm {
+                        connection.set(POM_SCM_CONNECTION)
+                        developerConnection.set(POM_SCM_CONNECTION)
+                        url.set(POM_URL)
                     }
-                    developer {
-                        id.set("cortinico")
-                        name.set("Nicola Corti")
-                        email.set("corti.nico@gmail.com")
+                    developers {
+                        developer {
+                            id.set("paolorotolo")
+                            name.set("Paolo Rotolo")
+                            email.set("rotolopao@gmail.com")
+                        }
+                        developer {
+                            id.set("cortinico")
+                            name.set("Nicola Corti")
+                            email.set("corti.nico@gmail.com")
+                        }
                     }
                 }
             }
         }
     }
+
+    repositories {
+        maven {
+            name = "OSSRH"
+            url = uri("https://s01.oss.sonatype.org/content/repositories/snapshots/")
```

Comment:
Heads up: this will work only for `-SNAPSHOT` releases.
Once we bump to a stable, this repository won't accept the release anymore.

I'd suggest we move to something like this:
https://github.com/gradle-nexus/publish-plugin